src/config/locales/fr/translation.json

```diff
@@ -0,0 +1,237 @@
+      "services": "Services",
+      "hide": "Cacher Whalebird",
+      "hide_others": "Cacher les autres",
+      "show_all": "Tout afficher",
+      "quit": "Quitter"
+    },
+    "toot": {
+      "name": "Pouet",
+      "new": "Nouveau"
+    },
+    "edit": {
+      "name": "Modifier",
+      "undo": "Défaire",
+      "redo": "Refaire",
+      "cut": "Couper",
+      "copy": "Copier",
+      "paste": "Coller",
+      "select_all": "Tout sélectionner"
+    },
+    "view": {
+      "name": "Afficher",
+      "toggle_full_screen": "Basculer en mode plein écran"
+    },
+    "window": {
+      "name": "Fenêtre",
+      "close": "Fermer la fenêtre",
+      "minimize": "Minimiser",
+      "jump_to": "Aller à"
+    }
+  },
+  "side_menu": {
+    "edit_profile": "Éditer mon profil",
+    "home": "Accueil",
+    "notification": "Notifications",
+    "favourite": "Favoris",
+    "local": "Fil public local",
+    "public": "Fil public global",
+    "hashtag": "Hashtag",
+    "search": "Rechercher",
+    "lists": "Listes"
+  },
+  "header_menu": {
+    "home": "Accueil",
+    "notification": "Notifications",
+    "favourite": "Favoris",
+    "local": "Fil public local",
+    "public": "Fil public global",
+    "hashtag": "Hashtag",
+    "search": "Rechercher",
+    "lists": "Listes",
+    "members": "Membres",
+    "filter": {
+      "title": "Filtrer",
+      "placeholder": "Filtrer par une expression régulière",
+      "apply": "Appliquer"
+    }
+  },
+  "preferences": {
+    "title": "Préférences",
+    "general": {
+      "title": "Général",
+      "appearance": "Apparence",
+      "theme_color": "Couleur du thème:",
+      "theme": {
+        "light": "Clair",
+        "dark": "Foncé"
+      },
+      "font_size": "Taille des caractères:",
+      "display_style": {
+        "title": "Format d'affichage:",
+        "display_name_and_username": "Nom et utilisateur⋅trice",
+        "display_name": "Nom",
+        "username": "Utilisateur⋅trice"
+      },
+      "toot": "Pouet",
+      "visibility": {
+        "title": "Visibilité par défaut:",
+        "public": "Public",
+        "unlisted": "Public sans être affiché sur le fil public",
+        "private": "Abonné⋅e⋅s uniquement",
+        "direct": "Message direct"
+      },
+      "sounds": "Sons",
+      "fav_rb_sound": "Partages et favoris:",
+      "toot_sound": "Pouets:"
+    },
+    "account": {
+      "title": "Compte",
+      "connected": "Compte connecté",
+      "username": "Utilisateur⋅trice",
+      "domain": "Domaine",
+      "association": "Association",
+      "order": "Ordre",
+      "remove_association": "Supprimer l'assocication",
+      "remove_all_associations": "Supprimer toutes les associations",
+      "confirm": "Confirmer",
+      "cancel": "Annuler",
+      "confirm_message": "Êtes-vous certain de vouloir supprimer toutes les associations ?"
+    },
+    "language": {
+      "title": "Langue",
+      "display_language": "Langue",
+      "notice": "Nécessite un redémarrage",
```

Comment:
__[Misspell]__

"marrage" is a misspelling of "marriage"


<!-- Comment by [Sider](https://sider.review) -->

Reply:
I guess it's a bot that made that suggestion ?  In any case, "redémarrage" is the correct French word for "restart".  "Marriage" means "wedding" ;-p

Reply:
Oh, sorry. It is a comment from auto linter. And may be it does not consider French. Please ignore the misspelling comments.